common/primitives/src/schema.rs

```diff
@@ -0,0 +1,5 @@
+use frame_support::BoundedVec;
+
+pub type SchemaId = u16;
```

Comment:
There is an existing `SchemaId` type defined in messages.rs. Please remove that and replace all of references with this one.